pkg/ccl/backupccl/targets.go

```diff
@@ -472,15 +473,17 @@ func MakeBackupTableEntry(
 		ind := -1
 		for i, b := range backupManifests {
 			if b.StartTime.Less(endTime) && endTime.LessEq(b.EndTime) {
-				if endTime != b.EndTime {
-					return BackupTableEntry{}, errors.Newf("should specify the exact backup time, next backup timestamp is %s", b.EndTime.AsOfSystemTime())
+				if endTime != b.EndTime && b.MVCCFilter != MVCCFilter_All {
+					return BackupTableEntry{}, errors.Newf(
```

Comment:
nit: this can be cleaned up by replacing `backupOptRevisionHistory` with %q and putting it as a format arg.